README.md

```diff
@@ -31,17 +31,47 @@ __gtk__ expects __GTK+__, __GLib__ and __Cairo__ development files to be install
 
 ### Windows
 
+Make sure you have the [GNU
+ABI](https://www.rust-lang.org/downloads.html#win-foot) version of the rust
+compiler installed.  Since we need our own toolchain make sure to uncheck
+"Linker and platform libraries" in the Rust setup.
+![Screenshot](rust_setup.png)
+
+If you're not sure, check if your Rust installation has `gcc.exe` and `ld.exe`
+in its `bin` directory, you may get a linking error `ld: cannot find -limm32`.
```

Comment:
This sentence seems slightly broken now. And I think it's now safe to say that the presence of those files will definitely cause the error.